dist/angular-patternfly.js

```diff
@@ -1670,6 +1683,7 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", "$window
        }
 
        $scope.data = {
+         dataAvailable: true,
```

Comment:
We added the flag in the angular patternfly directive so the application could differentiate between an empty dataset vs data unavailable.